doc/index.rst

```diff
@@ -92,6 +92,26 @@ both ``ComputedField`` and the other field class::
 Contributing
 ------------
 
-See the `contributing docs`_.
+End Users please see the `contributing docs`_.
 
-.. _contributing docs: https://github.com/orcasgit/django-computed-property/blob/master/CONTRIBUTING.rst
\ No newline at end of file
+.. _contributing docs: https://github.com/brechin/django-computed-property/blob/master/CONTRIBUTING.rst
+
+On top of the above, developers please consider the following:
+
+Reporting Issues or Suggestions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you see an issue with the library or have a request for new functionality, please file an issue in GitHub.
+
+Please include specifics about which version of Python you're using, as well as which version of the django-computed-property library you're using.
```

Comment:
Version of Django is also important, it would be good to also request that here. I just added it to `CONTRIBUTING.md`.

Reply:
merged into my branch.